app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SettingsFragment.java

```diff
@@ -49,4 +58,47 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
     public void changeHeadsUpEnabled(boolean enabled) {
         binding.headsUpSwitchRow.setEnabled(enabled);
     }
+
+    @Override
+    public void showLanguagesDialog() {
+        List<Locale> locales = LocaleUtil.SUPPORT_LANG;
+        List<String> languages = Observable.fromIterable(locales)
+                .map(locale -> getString(LocaleUtil.getLanguage(locale)))
+                .toList()
+                .blockingGet();
+
+        List<String> languageIds = Observable.fromIterable(locales)
```

Comment:
I did not know the existence of `com.annimon:stream:1.1.5`.
NEXT PR. `Observable` to `Stream`.